server/src/main.cpp

```diff
@@ -28,6 +36,17 @@ int main(int argc, char* argv[])
 
     ins_service::IndoorNavigationService ins;
     ins.Init(addr, thread_count);
+
+    // Register abort & terminate signals respectively
+    signal(SIGINT, kill_server);
```

Comment:
Should we not specify the std:: namespace in front of `signal`?

Reply:
Agreed, we should use std::signal